requirements.txt

```diff
@@ -1,2 +1 @@
 Cython==0.24.1
-numpy==1.11.1
```

Comment:
Why are you removing this dependency? `requirements.txt` is only used when setting up a virtual environment, which will only happen when someone clones the repo and runs tests and whatnot. They'll need numpy for that. This is separate from the numpy dependency handling in `setup.py`.